packages/neon_framework/test/validation_tile_test.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:neon_framework/l10n/localizations.dart';
+import 'package:neon_framework/src/theme/theme.dart';
+import 'package:neon_framework/src/widgets/validation_tile.dart';
+
+Widget wrapWidget(Widget widget, [TargetPlatform platform = TargetPlatform.android]) {
+  final theme = AppTheme.test(platform: platform);
+  const locale = Locale('en');
+
+  return MaterialApp(
+    theme: theme.lightTheme,
+    localizationsDelegates: NeonLocalizations.localizationsDelegates,
+    supportedLocales: NeonLocalizations.supportedLocales,
+    locale: locale,
+    home: widget,
+  );
+}
+
+void main() {
+  group('NeonValidationTile', () {
+    testWidgets('NeonValidationTile material', (widgetTester) async {
+      var state = ValidationState.loading;
+      var widget = Material(child: NeonValidationTile(title: 'title', state: state));
+      await widgetTester.pumpWidget(wrapWidget(widget));
```

Comment:
The  widget also doesn't need to be stored in a variable first since it is only used once.

Reply:
inlining the widget makes the test less readable